### PR TITLE
Messaging integration

### PR DESCRIPTION
**Pull request by Thurfjell**
Adds some new pseudo api endpoints to push messaging through the message package. 
Completing a payment and completing the workflow for  renew licence, will send a templated message (create new template if not exists).

Public servants  can send custom messages to users with a new form workflow in message app.

### CODE REVIEW

File: apps/messaging/app/[locale]/messages/[messageId]/page.tsx [hunk 1]
```diff
@@ -0,0 +1,32 @@
+import { api } from "messages";
+import Link from "next/link";
+import { notFound } from "next/navigation";
+
+export default async (props: { params: { messageId: string } }) => {
+  const message = await api.getMessage(props.params.messageId);
+  console.log({ message });
```
Comment (GiuliaTeggi) [inline apps/messaging/app/[locale]/messages/[messageId]/page.tsx:7]:
```suggestion
```

File: apps/messaging/app/[locale]/messages/[messageId]/page.tsx [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+  if (!message) {
+    // dunno what to show here :(
```
Comment (GiuliaTeggi) [inline apps/messaging/app/[locale]/messages/[messageId]/page.tsx:9]: Maybe also add a log here just to say 'message not found' for future dev reference when we further develop this?

File: apps/payments/app/[locale]/(hosted)/paymentRequest/bankTransfer/page.tsx [hunk 3]
```diff
@@ -55,6 +55,8 @@ async function getPaymentDetails(
     user_name: userInfo.user_name,
   };
 
+  console.log({ paymentDetails });
```
Comment (GiuliaTeggi) [inline apps/payments/app/[locale]/(hosted)/paymentRequest/bankTransfer/page.tsx:58]:
```suggestion
```

File: apps/web/app/[locale]/[event]/MyLifeEvents.tsx [hunk 4]
```diff
@@ -120,26 +122,62 @@ export default async () => {
...
+                  ).href
+                }
+              >
+                {msg.subject}
+              </Link>
+              <p className="govie-body" style={{ margin: "unset" }}>
+                {msg.content}
+              </p>
+              <hr className="govie-section-break govie-section-break--visible" />
+            </li>
```
Comment (GiuliaTeggi) [inline apps/web/app/[locale]/[event]/MyLifeEvents.tsx:178]: Could you please update this so to have the same styling of the 'eventsToRender' version? (e.g. including the > svg)

File: apps/web/tsconfig.json [hunk 5]
```diff
@@ -17,7 +17,8 @@
         "name": "next"
       }
     ],
-    "strictNullChecks": true
+    "strictNullChecks": true,
+    // "rootDirs": ["./web.d.ts"]
```
Comment (GiuliaTeggi) [inline apps/web/tsconfig.json:21]: Is this comment needed?

File: packages/auth/route.ts [hunk 6]
```diff
@@ -10,6 +10,8 @@ export default async function (req: Request) {
 
   const loginUrl = process.env.LOGIN_URL;
 
+  console.log("so fresh..");
```
Comment (GiuliaTeggi) [inline packages/auth/route.ts:13]:
```suggestion
```

File: packages/messages/migrations/20240318121837-simplication-poc.js [hunk 7]
```diff
@@ -0,0 +1,86 @@
...
+  //   created_at timestamptz default now()
+  // );
+
+  // create table message_interpolation_accessors(
+  //   message_id uuid not null,
+  //   value_accessor text not null,
+  //   key_accessor text not null
+  // );
+
+  // `);
```
Comment (GiuliaTeggi) [inline packages/messages/migrations/20240318121837-simplication-poc.js:73]: Do you want to keep these comments?

File: apps/web/app/[locale]/api/test/route.ts [hunk 8]
```diff
@@ -2,10 +2,15 @@ import { NextRequest } from "next/server";
 import { redirect } from "next/navigation";
 import { pgpool } from "../../../utils/postgres";
 import { driving } from "../../../utils/routes";
+import { api, temporaryMockUtils } from "messages";
+import { PgSessions } from "auth/sessions";
+
+export async function POST() {
+  console.log("hoho");
```
Comment (GiuliaTeggi) [inline apps/web/app/[locale]/api/test/route.ts:9]: 🎅 ??

**Review by GiuliaTeggi — COMMENTED** [review]
Looks good! just a few small comments

File: apps/messaging/app/[locale]/templates/Templates.tsx [hunk 9]
```diff
@@ -36,6 +36,21 @@ export default async () => {
...
+          href={"/test"}
+          style={{
+            margin: "unset",
+            paddingLeft: "12px",
+            width: "100%",
+            display: "flex",
+            justifyContent: "flex-start",
+          }}
+        >
+          Skicka bror
```
[the remark this answers is not in the document]
Comment (Thurfjell) [reply apps/messaging/app/[locale]/templates/Templates.tsx:51]: HAhahaha I think I'll just remove it. 😇

**Pull request merged**